Sources/StreamChat/Config/TokenProvider_Tests.swift

```diff
@@ -17,15 +17,16 @@ final class TokenProvider_Tests: StressTestCase {
 
     func test_developmentProvider_propagatesToken() throws {
         // Create a user identifier.
-        let userId: UserId = .unique
+        let userId: UserId = "luke_skywalker"
```

Comment:
Can we keep the `.unique` here and rather check only for the static parts of the token string explicitly?